app/stac_api/validators.py

```diff
@@ -408,25 +413,15 @@ def validate_item_properties_datetimes_dependencies(
             message = "Property start_datetime can't be null when no property datetime is given"
             logger.error(message)
             raise ValidationError(_(message), code='invalid')
-
-    if properties_datetime is None:
         if properties_end_datetime < properties_start_datetime:
             message = "Property end_datetime can't refer to a date earlier than property "\
             "start_datetime"
             raise ValidationError(_(message), code='invalid')
-
-
-def validate_item_properties_datetimes(
-    properties_datetime, properties_start_datetime, properties_end_datetime
-):
-    '''
-    Validate datetime values in the properties Item attributes
-    '''
-    validate_item_properties_datetimes_dependencies(
-        properties_datetime,
-        properties_start_datetime,
-        properties_end_datetime,
-    )
+        if properties_expires is not None:
+            if properties_expires < properties_end_datetime:
```

Comment:
maybe we should even enforce a minimal lifetime of an object, e.g. 24/72/... hours?

Reply:
As discussed offline, we are not sure about the different use cases, so for now we keep it simple and don't add more restrictions.